Description: Correct build sequence.
	When cross compiling the source, the specific programs built during
	the compilation phase are relevant for execution.
	Need to build those programs with the right way.

diff --git a/ht-2.1.0+repack1/debian/rules b/ht-2.1.0+repack1/debian/rules
index 92463cb..d7f90bf 100755
--- a/ht-2.1.0+repack1/debian/rules
+++ b/ht-2.1.0+repack1/debian/rules
@@ -3,6 +3,10 @@
 %:
 	dh $@
 
+override_dh_auto_build:
+	make CC="$(DEB_BUILD_GNU_TYPE)-gcc" -C tools
+	dh_auto_build
+
 override_dh_auto_install:
 	dh_auto_install
 	mv $(CURDIR)/debian/ht/usr/bin/ht $(CURDIR)/debian/ht/usr/bin/hte
